absl/strings/internal/str_format/checker.h

```diff
@@ -22,7 +22,7 @@
 // Compile time check support for entry points.
 
 #ifndef ABSL_INTERNAL_ENABLE_FORMAT_CHECKER
-#if ABSL_HAVE_ATTRIBUTE(enable_if) && !defined(__native_client__)
+#if ABSL_HAVE_ATTRIBUTE(enable_if) && !defined(__native_client__) && !defined(__INTELLISENSE__)
```

Comment:
Please leave a comment or a link to a bug so feature readers can figure out why we disabled this under intellisense.

Reply:
fixed